finality-aleph/src/party/backup.rs

```diff
@@ -41,8 +43,14 @@ impl From<io::Error> for BackupLoadError {
 
 impl std::error::Error for BackupLoadError {}
 
-pub type Saver = Box<dyn Write + Send + Sync>;
-pub type Loader = Box<dyn Read + Send + Sync>;
+pub trait BothRead: Read + AsyncRead {}
+impl<T: Read + AsyncRead> BothRead for T {}
+
+pub trait BothWrite: Write + AsyncWrite {}
```

Comment:
hmm naming is strange :P

Reply:
Yeah, these are here only because you cannot have something `dyn` with more than one real trait. I still cannot think of a better name. :<

Reply:
yrsh, me too :P

Reply:
meh